google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -110,37 +113,36 @@ public static void main(String[] args) {
    */
   private void runExample(
       GoogleAdsClient googleAdsClient, RemoveFeedItemAttributeValueParams params) {
-    updateFeedItem(googleAdsClient, params.customerId, params.feedId, params.feedItemId);
+    removeAttributeFromFeedItem(googleAdsClient, params);
```

Comment:
I know it makes the args list more verbose, but we generally pass the individual params attributes so users can easily copy/paste portions of the example without needing a dependency on the example's params class.